src/_component/about-me/about-me.scss

```diff
@@ -25,7 +25,7 @@
 button[class^="button__pill"] i[class^="icon__"] {
   width: 16px;
   height: 16px;
-  margin-right: 12px;
+  // margin-right: 12px;
```

Comment:
If you don't need this commented line for later you can delete it. It will look cleaner!